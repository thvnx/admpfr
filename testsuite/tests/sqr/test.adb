with Ada.Text_IO; use Ada.Text_IO;
with Admpfr;      use Admpfr;

procedure Test is
   M, N : Mpfloat;
begin
   N.Set (Float (1.0));

   M.Sqr (N);
   Put_Line (M'Image);

   N.Set (Float (2.0));

   M.Sqr (N);
   Put_Line (M'Image);
end Test;
