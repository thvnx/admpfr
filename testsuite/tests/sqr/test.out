1.0000000000000000e+00
4.0000000000000000e+00
